ResidentInformationApi/V1/Gateways/AcademyInformationGateWay.cs

```diff
@@ -12,19 +12,21 @@ namespace ResidentInformationApi.V1.Gateways
     public class AcademyInformationGateway : IAcademyInformationGateway
     {
         private readonly HttpClient _client;
-        private readonly string _baseUrl;
-
         public AcademyInformationGateway(HttpClient client)
         {
             _client = client;
-            _baseUrl = Environment.GetEnvironmentVariable("ACADEMY_API_ENDPOINT");
+            //_baseUrl = Environment.GetEnvironmentVariable("ACADEMY_API_ENDPOINT");
```

Comment:
If this isn't needed then you can delete this line

Reply:
yes